src/timeline/js/directives/ruler.js

```diff
@@ -95,6 +94,11 @@ App.directive("tlScrollableTracks", function () {
         element.removeClass("drag_cursor");
       });
 
+      element.on("scroll", function(){
```

Comment:
We do already have an .on("scroll" listener above that we could use, correct?

Reply:
You're right. I'll move that